extensions/fairifier-rdf-extension/deri/grefine/rdf/operations/LoadRdfSchemaOperation.java

```diff
@@ -0,0 +1,137 @@
+package org.deri.grefine.rdf.operations;
+
+import java.io.IOException;
+import java.io.LineNumberReader;
+import java.io.Writer;
+import java.util.Properties;
+
+import org.deri.grefine.rdf.RdfSchema;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.json.JSONWriter;
+
+import com.google.refine.history.Change;
+import com.google.refine.history.HistoryEntry;
+import com.google.refine.model.AbstractOperation;
+import com.google.refine.model.Project;
+import com.google.refine.operations.OperationRegistry;
+import com.google.refine.util.ParsingUtilities;
+import com.google.refine.util.Pool;
+
+public class LoadRdfSchemaOperation extends AbstractOperation {
+
+    final protected RdfSchema _schema;
+
+    public LoadRdfSchemaOperation(RdfSchema schema) {
+        this._schema = schema;
+    }
+
+    static public AbstractOperation reconstruct(Project project, JSONObject obj)
+            throws Exception {
+        return new LoadRdfSchemaOperation(RdfSchema.reconstruct(obj
+                .getJSONObject("schema")));
+    }
+
+    public void write(JSONWriter writer, Properties options)
+            throws JSONException {
+        writer.object();
+        writer.key("op");
+        writer.value(OperationRegistry.s_opClassToName.get(this.getClass()));
+        writer.key("description");
+        writer.value("Load RDF schema skeleton");
+        writer.key("schema");
+        _schema.write(writer, options);
+        writer.endObject();
+
+    }
+
+    @Override
+    protected String getBriefDescription(Project project) {
+        return "Load RDF schema skelton";
+    }
+
+    @Override
+    protected HistoryEntry createHistoryEntry(Project project,
+            long historyEntryID) throws Exception {
+        String description = "Load RDF schema skeleton";
+        
+        Change change = new RdfSchemaChange(_schema);
+        
+        return new HistoryEntry(historyEntryID, project, description,
+                LoadRdfSchemaOperation.this, change);
+    }
+
+    static public class RdfSchemaChange implements Change {
```

Comment:
This class is 100% identical to [the save equivalent](https://github.com/DTL-FAIRData/FAIRIFIER/blob/development/extensions/fairifier-rdf-extension/deri/grefine/rdf/operations/SaveRdfSchemaOperation.java#L64). There is zero reason for this copy to exist.

Reply:
The description is different in the class wich shows in the history provinence

Reply:
should i use inheritance maybe?

Reply:
https://www.diffchecker.com/epxRIvPe No difference in the class. So instead of copying the class, just reference it.

Reply:
ok